### PR TITLE
Remove hard coded module version in java code

### PR DESCRIPTION
**Pull request by chamil321**

### CODE REVIEW

File: mime-native/src/main/java/org/ballerinalang/mime/util/MimeUtil.java [hunk 1]
```diff
@@ -229,7 +230,7 @@ public static BObject parseMediaType(BObject mediaType, String contentType) {
     }
 
     public static void setMediaTypeToEntity(BObject entityStruct, String contentType) {
-        BObject mediaType = ValueCreator.createObjectValue(PROTOCOL_MIME_PKG_ID, MEDIA_TYPE);
+        BObject mediaType = ValueCreator.createObjectValue(MimeUtil.getMimePackage(), MEDIA_TYPE);
```
Comment (aashikam) [inline mime-native/src/main/java/org/ballerinalang/mime/util/MimeUtil.java:233]: Do we need `MimeUtil` here. This is inside `MimeUtil` right?

**Review by aashikam — APPROVED** [review]
LGTM

File: mime-native/src/main/java/org/ballerinalang/mime/util/MimeConstants.java [hunk 2]
```diff
@@ -34,13 +35,13 @@
     public static final String MEDIA_TYPE = "MediaType";
     public static final String CONTENT_DISPOSITION_STRUCT = "ContentDisposition";
     public static final String PROTOCOL_PACKAGE_MIME = BALLERINA_PACKAGE_PREFIX + "mime";
+    @Deprecated
     public static final String PROTOCOL_MIME_PKG_VERSION = "1.0.6";
```
Comment (aashikam) [inline mime-native/src/main/java/org/ballerinalang/mime/util/MimeConstants.java:39]: We can entirely remove this right?

Comment (chamil321) [reply mime-native/src/main/java/org/ballerinalang/mime/util/MimeConstants.java:39]: This constant has been used in HTTP also. So once the HTTP is migrated this constant will be removed. So for the moment marked as deprecated

**Pull request merged**